container/lxd/initialisation.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2016 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-// +build go1.3
+// +build go1.3, linux
```

Comment:
While I think that is fair, the only platform that LXD runs on is Linux, so perhaps what I should do is rename initialisation.go --> initialisation_linux.go and have initialisation.go do !linux?